docs/source/SMG/workflow.rst

```diff
@@ -0,0 +1,65 @@
+Developer Workflow
+=====================================
+
+Workflow
+^^^^^^^^^^^^^^^^^^^^^^
+
+#. Select a story from the waffle board.
```

Comment:
maybe a link to the waffle board here?

Reply:
Good idea, I'll update the story about adding a workflow page.